<?xml version="1.0"?>

<launch>
  <arg name="namespace" default="flir_lepton_sensor" />
  <arg name="gray_image" default="true" />
  <arg name="rgb_image" default="true" />

  <!-- Flir Lepton Node -->
  <node
    type="flir_lepton_sensor_node"
    name="flir_lepton_sensor"
    pkg="flir_lepton_sensor"
    respawn="false"
    ns="$(env ROS_BBB)"
    output="screen" >


    <!-- Load configuration parameters yaml -->
    <rosparam
      file="$(find flir_lepton_sensor)/config/params.yaml"
      command="load"
      ns="$(env ROS_BBB)"/>
  </node>

  <param name="$(arg namespace)/gray_image" value="$(arg gray_image)" />
  <param name="$(arg namespace)/rgb_image" value="$(arg rgb_image)" />

</launch>
